cli/plugins/test/index.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const pa11y = require('pa11y');
+
+class Test {
+  constructor(joy) {
+    const prog = joy.prog;
+    prog
+      .command('test', 'Run various tests including a11y and static analysis')
+      .argument('<subcommand>', 'subcommand [a11y, static]', ['a11y', 'static'])
+      .option('-u, --urls <urls>', 'URLs to test', prog.LIST)
+      .action(async (args, options, logger) => {
+        logger.debug("arguments: %j", args);
+        logger.debug("options: %j", options);
+        const code = await this[args.subcommand].call(joy, options, logger);
+        return code;
+      })
+      .argument('<x>', 'x', ['x'])
+      .option('-x, --xrls <xrls>', 'xRLs to test', prog.LIST)
+      .action(async (args, options, logger) => {
+        logger.debug("arguments: %j", args);
+        logger.debug("options: %j", options);
+        const code = await this[args.subcommand].call(joy, options, logger);
+        return code;
+      })
+
+  }
+
+  /**
+     * Wraps a11y module
+     * Todo: More research to see if this is the best module or test app to use
+     * 
+     * @param {*} args 
```

Comment:
This may be outdated based on the current parameters accepted

Reply:
Thanks, I'll go through the code comments. I use a simple plugin that stubs the parameters but doesn't track and update. Do either you or @simpleimpulse know of any code doc tools that adjust for changes? Wondering if there's a custom eslint rule we can employ and or write perhaps?